### ISSUE TITLE
Music Disappearing

### ISSUE THREAD

**Issue by honeyymiilk** [OMGDuke/SDH-GameThemeMusic#67]
So basically it’s just like it says the music disappears, the moment I go to open the option to select theme music, i’m stuck watching all my options slowly or quickly disappear before I get a chance to listen to any of them. No matter how many times I reload or reinstall the plugin I still have to deal with this issue, maybe i’m missing something? I’m not too sure this is my first time using plugins for my steam deck.

**Comment by atomfist** [comment]
I am running into this as well. I am on a fresh install of SteamOS and Decky

**Comment by anzryzen** [comment]
> I am running into this as well. I am on a fresh install of SteamOS and Decky

I think the lastest update might have broken something in it.

**Comment by richpanda1214** [comment]
Also have this issue. Plugin is completely broken.

**Comment by FlyinPancake** [comment]
Same issue here

**Comment by ThisByzantineConduit** [comment]
This has been happening for a while, but I've been able to circumvent it by choosing an alternate Piped Instance until now. For years this plugin worked flawlessly but now the default/official international Piped Instance and all alternative U.S.A. Piped Instances, and many from other regions, populate a big list of songs which then completely disappears leaving no music options.

This is consistent no matter how generalized my search terms are. Any idea on a fix? Tried reinstalling the plugin to no avail.
